R/staggregate.R

```diff
@@ -654,13 +871,13 @@ staggregate_bin <- function(data, overlay_weights, daily_agg, time_agg = "month"
 
   # Function check_bins to determine which bins data points fall into
   check_bins <- function(x){
-    clim_daily_table <- raster::values(clim_daily)
+    clim_daily_table <- terra::values(clim_daily)
```

Comment:
Why is this being done here and above? It looks like this is done above, and doing it here means it is done each time in the lapply below. So its either one extra time or many extra times